app/messages.py

```diff
@@ -333,6 +338,12 @@
 }
 
 UNEXPECTED_INPUT = {"message": "Unexpected input is detected. Please check your input to make sure only approved fields are to be submitted."}
-USER_ID_IS_NOT_RETRIEVED = {
+USER_ID_IS_NOT_RETRIEVED_WITH_GET_USER = {
     "message": "You must view your personal details first before you can proceed."
 }
+USER_ID_IS_NOT_RETRIEVED_WITH_GET_ORGANIZATION = {
+    "message": "You must send GET /organization request first before you can proceed."
+}
+IS_ORGANIZATION_REP_NOT_DECLARED = {
+    "message": "You have not declared that you are representing an organization. You can do this on My Space > Additional Info."
```

Comment:
I think remove the You can do this. It's really upto change, if we change any structure on the UI, we'll always have to remember to change this text. 
Too tightly coupled